extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -354,6 +342,20 @@ export class AmpAnalytics extends AMP.BaseElement {
         this.config_['transport'], ampAdResourceId);
   }
 
+  /**
+   * Gets the resourceID of the parent amp-ad element.
+   * Throws an exception if no such element.
+   * @returns {string}
+   * @VisibleForTesting
+   */
+  assertAmpAdResourceId() {
+    const TAG = this.getName_();
```

Comment:
Nit: I would just inline this.getName_() below rather than creating new variable